tests/integration/tables/windows_firewall_rules.cpp

```diff
@@ -0,0 +1,242 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+// Sanity check integration test for connectivity
+// Spec file: specs/windows/windows_firewall_rules.table
+
+#include <osquery/tests/integration/tables/helper.h>
+
+#include <osquery/core/core.h>
+#include <osquery/core/tables.h>
+#include <osquery/tables/networking/windows/windows_firewall_rules.h>
+
+namespace osquery {
+
+namespace tables {
+extern QueryData renderFirewallRules(const FirewallRules& rules);
```

Comment:
This function should be exported from the public header

Reply:
done